game.py

```diff
@@ -20,7 +21,36 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+    
+    
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+
+    wrong_guesses_list = []
+    
+    while (len(wrong_guesses_list) < SNOWMAN_MAX_WRONG_GUESSES 
+           and not is_word_guessed(snowman_word, correct_letter_guess_statuses)): 
```

Comment:
Love the formatting here! By breaking your `while` statement into two separate lines, you ensure that your code doesn't go beyond the conventional 80 character limit per line.